advisories/github-reviewed/2025/06/GHSA-x5gf-qvw8-r2rm/GHSA-x5gf-qvw8-r2rm.json

```diff
@@ -1,21 +1,17 @@
 {
   "schema_version": "1.4.0",
   "id": "GHSA-x5gf-qvw8-r2rm",
-  "modified": "2025-07-10T19:21:01Z",
+  "modified": "2025-07-10T19:21:05Z",
   "published": "2025-06-09T21:30:51Z",
   "aliases": [
     "CVE-2025-5891"
   ],
   "summary": "pm2 Regular Expression Denial of Service vulnerability",
   "details": "A vulnerability classified as problematic was found in Unitech pm2 up to 6.0.6. This vulnerability affects unknown code of the file /lib/tools/Config.js. The manipulation leads to inefficient regular expression complexity. The attack can be initiated remotely. The exploit has been disclosed to the public and may be used.",
   "severity": [
-    {
-      "type": "CVSS_V3",
-      "score": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:L"
-    },
     {
       "type": "CVSS_V4",
-      "score": "CVSS:4.0/AV:N/AC:L/AT:N/PR:L/UI:N/VC:N/VI:N/VA:L/SC:N/SI:N/SA:N/E:P/CR:X/IR:X/AR:X/MAV:X/MAC:X/MAT:X/MPR:X/MUI:X/MVC:X/MVI:X/MVA:X/MSC:X/MSI:X/MSA:X/S:X/AU:X/R:X/V:X/RE:X/U:X"
```

Comment:
I didn't intend to remove these metrics, but the advisory improvement screen forced me to remove them, and I don't see a way to put them back. I've opened an issue about this limitation in the advisory improvement screen: https://github.com/github/advisory-database/issues/5357.